CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Compatibility
  - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
  - iPhone 5s devices and later with iOS 11+ is required.
+ 
+**Note:** Support for ESR 5.9 has ended and we strongly recommend upgrading to Extended Support Release 5.19 or a later version for improved security, performance, mobile app compatibility, and user experience. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details.
```

Comment:
We should note this is a **server** ESR/upgrade we are referring to. Since this is the mobile changelog, we should make sure the consequences of not upgrading are clear here. Should we add in that line from the blog post?

Reply:
We should say "required" not "highly recommended"

Reply:
Also move this to the first paragraph under the compatibility heading

Reply:
With bold title "Upgrade required to server version v5.19 or later"

Reply:
With bold title "Upgrade required to server version v5.19 or later"